presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -55,22 +55,6 @@ protected AbstractTestDistributedQueries(QueryRunner queryRunner, Session sample
         super(queryRunner, sampledSession);
     }
 
-    private void assertCreateTable(String table, @Language("SQL") String query, @Language("SQL") String rowCountQuery)
```

Comment:
Not a big deal, but try to avoid renaming and moving methods in the same commit (especially since the move is unrelated to this change), as it makes it harder to see if there are meaningful changes. What I usually do in these cases is introduce another commit before or after to do some "cleanup".